jpsreport/methods/Method_J.cpp

```diff
@@ -194,8 +194,12 @@ bool Method_J::Process(
                     }
                 } else {
                     for(int i = 0; i < (int) IdInFrame.size(); i++) {
-                        std::cout << XInFrame[i] * CMtoM << "   " << YInFrame[i] * CMtoM << "   "
-                                  << IdInFrame[i] << "\n";
+                        LOG_WARNING(
```

Comment:
I suggest to add text here: "Polygons could not be obtained for..."

Reply:
changed